core/src/main/java/io/aiven/klaw/controller/TopicController.java

```diff
@@ -91,19 +91,30 @@ public ResponseEntity<Map<String, String>> getTopicTeam(
         topicControllerService.getTopicTeamOnly(topicName, patternType), HttpStatus.OK);
   }
 
-  /*
-     For executing topic requests
-  */
+  /**
+   * @param pageNo Which page would you like returned e.g. 1
+   * @param currentPage Which Page are you currently on e.g. 1
+   * @param requestsType What type of requests are you looking for e.g. 'created' or 'deleted'
+   * @param teamName The name of the team that that you wish to filter the results by, e.g.
```

Comment:
Can we pass teamId from FE ?

Reply:
I will check with FE on this

Reply:
Outcome here is FE will use teamId either an existing API or we will create a new streamlined one for their use.